site/content/examples/09-transitions/04-custom-js-transitions/App.svelte

```diff
@@ -12,7 +12,7 @@
 		}
 
 		const text = node.textContent;
-		const duration = text.length * speed;
+		const duration = text.length / (speed * 0.01) ;
```

Comment:
what happens if the `speed` is 0 then?
should there be some minimum speed?

Reply:
If the speed is 0, then the text doesn't show up at all. That's also my interpretation of how it should behave: a stopped car never goes anywhere.

Given that it's an example and not production ready code, it seems to me it doesn't need a minimum value, as long as the default is not 0.

Reply:
```suggestion
		const duration = text.length / (speed * 0.01);
```